nav2_bringup/README.md

```diff
@@ -2,121 +2,86 @@
 
 The `nav2_bringup` package is an example bringup system for navigation2 applications.
 
-Notes: (December 2018, Crystal Release)
-* We recommend doing this on a Ubuntu 18.04 installation. We’re currently having build issues on 16.04. (see https://github.com/ros-planning/navigation2/issues/353)
+Notes: (June 2019, Dashing Release)
+* We recommend doing this on a Ubuntu 18.04 installation. We have build issues on 16.04. (see https://github.com/ros-planning/navigation2/issues/353)
 * This stack and ROS2 are still in heavy development and there are some bugs and stability issues being worked on, so please do not try this on a robot without taking *heavy* safety precautions. THE ROBOT MAY CRASH!
 * It is recommended to start with simulation using Gazebo before proceeding to run on a physical robot
 
 Install and build our code by following this guide:
 https://github.com/ros-planning/navigation2/blob/master/doc/BUILD.md
 
-## Launch Navigation2 in simulation with Gazebo (first time users)
+## Launch Navigation2 in simulation with Gazebo
 ### Pre-requisites:
 * Gazebo installed on the system
-* gazebo_ros_pkgs for ROS2 installed on the system
+* gazebo\_ros\_pkgs for ROS2 installed on the system
 * A Gazebo world for simulating the robot (see Gazebo tutorials)
 * A map of that world saved to a map.pgm and map.yaml (see ROS Navigation tutorials)
 
-### Terminal 1: Launch Gazebo and Rviz2
+### Terminal 1: Launch Gazebo
 
 Example: See [turtlebot3_gazebo models](https://github.com/ROBOTIS-GIT/turtlebot3_simulations/tree/ros2/turtlebot3_gazebo/models) for details
+
 ```
 export GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:<full/path/to/my_robot/models>
-ros2 launch nav2_bringup gazebo_rviz2_launch.py world:=<full/path/to/gazebo.world>
+gazebo --verbose -s libgazebo_ros_init.so <full/path/to/my_gazebo.world>
 ```
 
 ### Terminal 2: Launch your robot specific transforms
 
 Example: See [turtlebot3_gazebo](https://github.com/ROBOTIS-GIT/turtlebot3_simulations/tree/ros2/turtlebot3_gazebo) for details
 
-`ros2 launch turtlebot3_bringup burger_state_publisher.launch.py`
-
-### Terminal 3: Launch map_server and AMCL
-
 ```
-# Set the tf publisher node to use simulation time or AMCL won't get the transforms correctly
-ros2 param set /robot_state_publisher use_sim_time True
-# Launch map_server and AMCL, set map_type as "occupancy" by default.
-ros2 launch nav2_bringup nav2_bringup_1st_launch.py map:=<full/path/to/map.yaml> map_type:=occupancy use_sim_time:=True
+source turtlebot3_ws/install/setup.bash
+export TURTLEBOT3_MODEL=waffle
```

Comment:
`waffle` or `burger`

Reply:
waffle :)

Seriously, that's the physical robot we have been using and the parameters should be set for it (robot radius, etc).